requirements/base.rqr

```diff
@@ -4,3 +4,4 @@ mecab-python3
 absl-py
 dpath
 ipadic
+pydantic==2.6.0
```

Comment:
I think  should move this to a new requirements/service.rqr file, because people who don't need the service, should be be required to use it.

Reply:
yeah that's what @elronbandel wrote too. the thing is that when i try to move it i encounter an issue i mentioned above: https://github.com/IBM/unitxt/pull/568#issuecomment-1945599423
any suggestion how to workaround that?

Reply:
Removed this requirement, applied the solution described [here](https://github.com/IBM/unitxt/pull/568#issuecomment-1951159347).